core/iwasm/aot/aot_runtime.h

```diff
@@ -139,8 +142,8 @@ typedef struct AOTModule {
     AOTTableInitData **table_init_data_list;
 
     /* function type info */
-    uint32 func_type_count;
-    AOTFuncType **func_types;
+    uint32 type_count;
+    AOTFuncType **types;
```

Comment:
We will change `AOTFuncType` to `AOTType` in the future, right?

Reply:
Yes, I'll do it in next PR